src/en/space-station-14/round-flow/proposals/station-ai.md

```diff
@@ -0,0 +1,162 @@
+- Comms console - Separate and cheaper than others as it’s rather important and relatively low impact (except round stalling, oops?). Though would it be better to keep it MORE expensive?
+- Assorted consoles - Just a general category but they each should not be free.
+- Bolting a door - it’s an override in case of an emergency, not a go-to that can be spammed with impunity. Bolting off giant hallways would require managing resources, and making sure to unbolt things elsewhere. Keeping things bolted is possible, but not unless you the player wants to do nothing else.
+- Research - a resource sink that takes a lot of processing capability of the AI in both of the available resources at a base, and can be increased further (powers of two? exponential?) to provide more research. Should not be a way to completely ignore science department gameplay, but rather be a side option, requiring substantial resource investment to provide good enough amount of points.
+- Cargo bounty - Call it “Bluespace Calculation” or any other fancy name. Basically the same as research but also gives out something that can fulfil a cargo bounty. Giving it to cargo could be done through integration with the [paperwork management UI](#assorted-ideas)? Should be separate from research resource sink to not let AI just double-dip.
+
+#### Routines (Momentary actions)
+
+Things that don’t require sustained cost, but should not be completely free. Practically anything that interacts with things beyond a digital interface.
+
+- Opening a door! Minimal amount required.
+- Speech when the radio process is inactive. A somewhat substantial amount when surviving off a single or two AI servers, but not extremely hard hitting. Binary channel is always free however.
+- Left-click or alt-click on vents to unlock them, for cheap
+- **TODO**. Ideas: sensor sweep? Mass applying (un)bolting to everything an APC’s connected to? Atmospherics reset action that can be used from the action bar/alt-click radial menu and applied to an atmos alarm to reset it to default? Clicking vents to unlock them?
+- Consoles! They aren’t free and cut a decent chunk, to encourage the AI player to choose a Compute option, but in a pinch this should work too. It also gets paid by the player having to manually locate a console and being unable to check on things elsewhere but the vicinity of the console.
+
+### Sources
+
+- AI specialized machinery, contributing a significant amount of processing capability to the AI.
+  - The downside to this is that it requires someone physical to build it, and needs a decent amount of resources
+  - Such machinery should be difficult to destroy or even move. It stays where it stays, unless the situation requires it and someone puts enough effort to move it; or break it.
```

Comment:
It can easily just weigh two metric tons, same as AI core, to avoid being able to move it easily. You should be able to move it to rearrange a room but moving it across the station should be painful. I believe the solution to the AI dragging their core into maints and hiding it (to activate the doomsday device willy nilly) was the pinpointer pointing to them?